src/mui/list/FilterForm.js

```diff
@@ -113,6 +114,24 @@ FilterForm.defaultProps = {
 
 const enhance = compose(
     translate,
+    withProps(({ initialValues, filters }) => ({
```

Comment:
I don't understand this one, could you explain the rationale?

Reply:
Merge `initialialValues` passed by the `List` component (which may have read them from the querystring) with the defaultValues of all `alwaysOn` filters which have a `defaultValue` specified.

Reply:
`initialialValues` should probably be merged at the end.. back to WIP :)

Reply:
I confirm that this would have been a prb. Fixed it